spec/features/consumer/confirm_invitation_spec.rb

```diff
@@ -19,7 +19,7 @@
       visit spree.spree_user_confirmation_url(confirmation_token: user.confirmation_token)
 
       expect(user.reload.confirmed?).to be true
-      expect(page).to have_text "Change my password"
```

Comment:
The page was rendering `Change My Password` as the result of the missing translation thus, not matching this assertion.